spec/Section 3 -- Type System.md

```diff
@@ -2164,3 +2172,104 @@ to the relevant IETF specification.
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
 ```
+
+### @defer
+
+```graphql
+directive @defer(
+  if: Boolean! = true
+  label: String
+) on FRAGMENT_SPREAD | INLINE_FRAGMENT
+```
+
+The `@defer` directive may be provided on a fragment spread or inline fragment
+to indicate that execution of the related selection set should be deferred. When
+a request includes the `@defer` directive, the result may consist of multiple
+responses: the initial response containing all non-deferred data, while
+subsequent responses include deferred data.
```

Comment:
I believe that we've refined the terminology around this: there's one _response_. The response for stream/defer is a stream consisting of an initial _result payload_ followed by a number of _incremental payloads_. I couldn't find where we discussed this, so please correct as appropriate.

```suggestion
payloads: the initial payload containing all non-deferred data, while subsequent
payloads include deferred data.
```

Reply:
@benjie after [the last discussion we had](https://github.com/graphql/graphql-spec/pull/1124#issuecomment-2569497795), I decided to drop `payload`. The response section now says:

> The result of a GraphQL request must be either a single initial response or an
incremental stream. The response will be an incremental stream when the GraphQL
service has deferred or streamed data as a result of the `@defer` or `@stream`
directives. When the result of the GraphQL operation is an incremental stream,
the first value will be an initial response, followed by one or more subsequent
responses.